xl_auth/commands.py

```diff
@@ -608,3 +609,11 @@ def _get_manually_deleted_permissions():
                   % (permission.user.email, permission.collection.code, wipe_permissions))
             if wipe_permissions:
                 permission.delete()
+
+
+@click.command()
+def prod_run():
+    from sh import Command
+    flask_cmd = Command('flask')
+    click.echo(flask_cmd('db', 'upgrade').stderr)
```

Comment:
@mxtthias What happens when upgrade fails, for whatever reason? I mean, will it fail *after* printing the stderr output?

Reply:
Trying it out with the following raises `sh.ErrorReturnCode_2` and prints `/bin/ls: cannot access 'bad_file_name': No such file or directory` to STDERR:

```python
@click.command()
def prod_run():
    from sh import Command
    ls = Command('ls')
    click.echo(ls('bad_file_name'))
    flask_cmd = Command('flask')
    click.echo(flask_cmd('db', 'upgrade').stderr)
    execlp('gunicorn', 'gunicorn', '-c', 'gunicorn_conf.py', 'autoapp:app')
```

From this I would expect any non-successful run of `flask db upgrade` would raise some type of error. Should I investigate further?